nano/lib/rpcconfig.cpp

```diff
@@ -233,10 +233,21 @@ nano::error read_rpc_config_toml (boost::filesystem::path const & data_path_a, n
 	// Parse and deserialize
 	nano::tomlconfig toml;
 
+	std::stringstream config_overrides_stream;
+	for (auto const & entry : config_overrides)
+	{
+		config_overrides_stream << entry << std::endl;
+	}
+	config_overrides_stream << std::endl;
+
 	// Make sure we don't create an empty toml file if it doesn't exist. Running without a toml file is the default.
 	if (!error && boost::filesystem::exists (toml_config_path))
 	{
-		error = toml.read (toml_config_path);
+		error = toml.read (config_overrides_stream, toml_config_path);
+	}
+	else if (!error)
+	{
+		toml.read (config_overrides_stream);
```

Comment:
I wonder if this should be moved into a single block, so `!error` isn't repeated, i.e:
```
if (!error)
{
    if (boost::filesystem::exists (toml_config_path))
    {
        error = toml.read (config_overrides_stream, toml_config_path);
    }
    else
    {
        toml.read (config_overrides_stream);
    }
}
```
But is up to you.